scripts/reset-test-db.sh

```diff
@@ -60,19 +80,21 @@ else
     # Clean the cache directory
     rm -rf $DB_CACHE_DIR && mkdir -p $DB_CACHE_DIR
 
-    # Re-run migrations on the test database
-    echo "Issuing a migrate command to the bok_choy MySQL database for the lms django apps."
-    ./manage.py lms --settings bok_choy migrate --traceback --noinput
-    echo "Issuing a migrate command to the bok_choy MySQL database for the cms django apps."
-    ./manage.py cms --settings bok_choy migrate --traceback --noinput
-
-    # Dump the schema and data to the cache
-    echo "Using the dumpdata command to save the fixture data to the filesystem."
-    ./manage.py lms --settings bok_choy dumpdata > $DB_CACHE_DIR/bok_choy_data.json
-    # dump_data does not dump the django_migrations table so we do it separately.
-    echo "Saving the django_migrations table of the bok_choy DB to the filesystem."
-    mysqldump -u root --no-create-info edxtest django_migrations > $DB_CACHE_DIR/bok_choy_migrations_data.sql
-    echo "Saving the schema of the bok_choy DB to the filesystem."
-    mysqldump -u root --no-data --skip-comments --skip-dump-date edxtest > $DB_CACHE_DIR/bok_choy_schema.sql
-fi
+    for db in "${database_order[@]}"; do
+        # Re-run migrations on the test database
+        echo "Issuing a migrate command to the bok_choy MySQL database for the lms django apps."
+        ./manage.py lms --settings bok_choy migrate --database $db --traceback --noinput
+        echo "Issuing a migrate command to the bok_choy MySQL database for the cms django apps."
+        ./manage.py cms --settings bok_choy migrate --database $db --traceback --noinput
```

Comment:
I did it both ways, I preferred reading the output where the DB alias was output on each line, so that's what I pushed.